mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -600,15 +604,21 @@ public void showUploadImageProgressbar(boolean b) {
         }
     }
 
+    public void urlBuilder() {
+        url = new StringBuilder();
+        url.append(PrefManager.getInstanceUrl()
+                + "clients/"
+                + "clientId"
+                + "/images?maxHeight=120&maxWidth=120");
+        String tempUrlCopy = url.toString();
+        tokenPositionStart = tempUrlCopy.indexOf("clientId");
+        tokenPositionEnd = tokenPositionStart + 8;
```

Comment:
ok @ishan1604 I will but can you help in figuring out why the checks are not passing on  my pr as I have only changed the way "url" is constructed as compared to my previous PR.
Thank you